src/components/chat/ChatContent.tsx

```diff
@@ -0,0 +1,90 @@
+import styled from 'styled-components';
+import { useRecoilValue } from 'recoil';
+import { useRef, useEffect } from 'react';
+import ChatItem from './ChatItem';
+import { IChatRoom, IUser, IMessage, IRoomId } from '../../store/interface';
+import { roomList, userInfo, roomInfo } from '../../store/atom';
+function ChatContent({ roomid }: IRoomId): JSX.Element {
+	const roomLists = useRecoilValue<IChatRoom[]>(roomList);
+	const roomId = useRecoilValue<Number>(roomInfo);
+	const roomIndex = roomLists.findIndex((room) => room.roomid === roomId);
+	const currentUser = useRecoilValue<IUser>(userInfo);
+	const chatList: IMessage[] = roomLists[roomIndex]?.messages || [];
+	const chatContent = useRef<HTMLDivElement>(null);
+	const prevTimeValue = useRef<String | null>(null);
+	let prevUserValue = -1;
+
+	const scrollChat = () => {
+		if (chatContent.current) {
+			chatContent.current.scrollTop = chatContent.current.scrollHeight;
+		}
+	}
+
+	const makeTime = (time: number) => {
+		const newTime = new Date(time);
+		let hours = newTime.getHours();
+		let minutes = newTime.getMinutes();
+		let hourFormat = hours < 12 ? `오전 ${hours}` : `오후 ${hours - 12}`;
+		let minFormat = minutes < 10 ? `0${minutes}` : minutes.toString();
+		const madeTime = hourFormat + ":" + minFormat;
+		return madeTime;
+	}
+
+	useEffect(() => {
+		scrollChat();
+	}, [roomLists]);
+
+	useEffect(() => {
+		scrollChat();
+	}, [currentUser]);

```

Comment:
```suggestion
	useEffect(() => {
		scrollChat();
	}, [roomLists, currentUser]);
```

요런 식으로 합쳐서 작성할 수 있습니다!